website/source/docs/guides/backup.html.md

```diff
@@ -0,0 +1,87 @@
+
+## Create Your First Backup
+
+The `snapshot save` command for backing up the cluster state has many configuration options. In a production environment, you will want to configure ACL tokens and client certificates for security. The configuration options also allow you to specify the datacenter and server to collect the backup data. Below are several examples. 
+ 
+First, we will run the basic snapshot command with the all the defaults, including `consistent` mode.
+
+```sh
+$ consul snapshot save backup.snap
+Saved and verified snapshot to index 1176
+```
+The backup will be saved locally in the directory where we ran the command. 
+
+You can view metadata about the backup with the `inspect` subcommand. 
+
+```sh
+$ consul snapshot inspect backup.snap
+ID           2-1182-1542056499724
+Size         4115
+Index        1182
+Term         2
+Version      1
+```
+
+To understand each field review the inspect [documentation](https://www.consul.io/docs/commands/snapshot/inspect.html). Notably, the `Version` field does not correspond to the version of the data. Rather it is the snapshot format version. 
+
+Next, let’s collect the cluster data from a non-leader by specifying stale mode.
+
+```sh
+$ consul snapshot save -stale backup.snap
+Saved and verified snapshot to index 2276
+```
+
+Once ACLs and agent certificates are configured, they can be passed in with the following flags.
+
+```sh
+$ consul snapshot save -stale -token=<ead6787-23thbd-666789> -ca-file=</path/to/file> backup.snap
+Saved and verified snapshot to index 2287
+```
+
+Alternatively, you can use environment variables to say the ACL token and CA cert. 
+
+For production use, the  `snapshot save` command or [API](https://www.consul.io/api/snapshot.html) should be scripted and run frequently. In addition to frequently backing up the cluster state, there are several use cases when you would also want to manually execute `snapshot save`. First, it should be best practice to backup the cluster before upgrading. If the upgrade does not go according to plan, this will ensure you have the latest data. Second, if the cluster loses quorum it may be beneficial to save the state before the non-leaders become divergent. Finally, you can manually snapshot the cluster for use of bootstrapping a new cluster. 
+
+Operationally, the backup process does not need to be executed on every server. Additionally, you can use the configuration options to save the backups to a mounted filesystem. The mounted filesystem can even be cloud storage, such as Amazon S3. The enterprise command `snapshot agent` automates this process.
+
+## Restore from Backup 
+
+Running the `restore` process should be straightforward. However, there are a couple actions you can take to ensure the process goes smoothly. First, make sure the cluster you are restoring is stable. Second, you will only need to run the process once, on the leader. The cluster gossip protocol will ensure the data is propagated to the non-leaders.  
```

Comment:
```suggestion
Running the `restore` process should be straightforward. However, there are a couple of actions you can take to ensure the process goes smoothly. First, make sure the cluster you are restoring is stable and has a leader. You can see this using `consul operator raft list-peers` and checking server logs and telemetry for signs of leader elections or network issues. You will only need to run the process once, on the leader. The Raft consensus protocol ensures that all servers restore the same state.
```

Reply:
I like that you added the information about checking for a leader!